guidelines.md

```diff
@@ -94,7 +94,7 @@ Fields: Proteomics, ProteoGenomics
 
 Name: The mass spectra are provided in the HUPO-PSI standard file format mzML.  
 Category: Gold   
-Description:
+Description: mzML is a HUPO-PSI data format that store mass spectrometry data (e.g mass spectra, chromatograms) using controlled vocabulary terms. mzML, is a well tested open-source and XML-based format.   
```

Comment:
We could even add the link to the mzML spec / website here. Markdown syntax is fully supported by the viewer for description and examples.